mat64/dense.go

```diff
@@ -306,6 +339,9 @@ func (m *Dense) Clone(a Matrix) {
 	m.mat = mat
 }
 
+// Copy makes a copy of elements of a into the receiver.
```

Comment:
Explain that it is similar to the copy built-in; it copies as much as the overlap between the two matrices and returns the number of rows and columns it copied.